package.json

```diff
@@ -2125,6 +2125,556 @@
+														"name": "Backward up expression",
+														"type": "command",
+														"command": "paredit.backwardUpSexp"
+													},
+													{
+														"key": "J",
+														"name": "Join expression",
+														"type": "command",
+														"command": "paredit.joinSexp"
+													},
+													{
+														"key": "L",
+														"name": "Forward up expression",
+														"type": "command",
+														"command": "paredit.forwardUpSexp"
+													},
+													{
+														"key": "S",
+														"name": "Slurp expression backward",
+														"type": "command",
+														"command": "paredit.slurpSexpBackward"
+													},
+													{
+														"key": "W",
+														"name": "Transpose expression",
+														"type": "command",
+														"command": "paredit.transpose"
+													}
+												]
+											},
+											{
+												"key": "m",
+												"name": "+Manage REPL session",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": ".",
+														"name": "Connect or jack-in",
+														"type": "command",
+														"command": "calva.jackInOrConnect"
+													},
+													{
+														"key": "c",
+														"name": "Connect to REPL server for project",
+														"type": "command",
+														"command": "calva.connect"
+													},
+													{
+														"key": "j",
+														"name": "Start REPL server for project (jack-in)",
+														"type": "command",
+														"command": "calva.jackIn"
+													},
+													{
+														"key": "r",
+														"name": "Refresh changed namespaces",
+														"type": "command",
+														"command": "calva.refresh"
+													},
+													{
+														"key": "s",
+														"name": "Select cljs build connection",
+														"type": "command",
+														"command": "calva.switchCljsBuild"
+													},
+													{
+														"key": "t",
+														"name": "Toggle cljc session (clj, cljs)",
+														"type": "command",
+														"command": "calva.toggleCLJCSession"
+													},
+													{
+														"key": "q",
+														"name": "Disconnect (quit) from REPL server",
+														"type": "command",
+														"command": "calva.disconnect"
+													},
+													{
+														"key": "C",
+														"name": "Run custom REPL command",
+														"type": "command",
+														"command": "calva.runCustomREPLCommand"
+													},
+													{
+														"key": "R",
+														"name": "Refresh all namespaces",
+														"type": "command",
+														"command": "calva.refreshAll"
+													}
+												]
+											},
+											{
+												"key": "r",
+												"name": "+Refactor (Calva 2.0.137 or greater)",
```

Comment:
Do we need to call out in the menu? Is there a reason people would be on a lower version? Can we document it instead of putting in the menu name?

Reply:
If we really need to put it, I propose something like `(Calva >= 2.0.137)`

Reply:
Calva refactor commands are only available from that specific version.  This is a very new feature that is not particularly memory efficient (it introduces clojure-lsp which currently has very high memory usage when running.  This is being addressed, but can cause an issue for those without lots of extra memory).

I could remove the refactor key bindings if its an issue, adding them back when these issues with Calva have been resolved.

Reply:
Why? Leave them here :) I think Steven is just talking about an aesthetic factor.
We have to choose between writing  `(Calva >= 2.0.137)` there or removing the parenthesis and documenting what you have written in the previous comment in the website. (we can do also both)

Reply:
> I think Steven is just talking about an aesthetic factor.

Right I don't think there is any issue with the bindings, only the name. I prefer to take out the ` (Calva 2.0.137 or greater)` and document it in https://vspacecode.github.io/docs/major-mode.

The reasons are 
1. If the commands doesn't exist, the user will get an error indicating the command doesn't exists. They will be able to look at the documentation website. (The experience is a bit like a user running major mode without installing the extension)
2. Since Calva is addressing the slowness, it will be improved in the future without us changing the name of the bindings. (We probably only need to update the documentation website)
3. The name looks a bit odd compare to all other

Reply:
Agreed.  I have removed the comment from the key binding menu name